cfme/tests/cloud_infra_common/test_retirement.py

```diff
@@ -294,3 +296,27 @@ def test_resume_retired_instance(retire_vm, provider, remove_date):
 
     verify_retirement_date(retire_vm, expected_date=retire_date if retire_date else 'Never')
     assert retire_vm.is_retired is False
+
+
+@pytest.mark.manual('manualonly')
```

Comment:
We can't automate this?

Reply:
good point, removing. everything in my FA is gonna be automated.